repository/postgres.go

```diff
@@ -157,12 +159,19 @@ func (p *postgres) SaveServiceRequest(serviceReq models.ServiceRequest) (models.
 	return pgServReq.ToServiceRequest(), err
 }
 
-func (p *postgres) GetWorkflows(pageNumber int, pageSize int) ([]models.Workflow, error) {
+func (p *postgres) GetWorkflows(pageNumber int, pageSize int, filters map[string]string) ([]models.Workflow, error) {
 	var pgWorkflows []models.PGWorkflow
-	err := p.getDb().Model(&pgWorkflows).
-		Limit(pageSize).
-		Offset(pageNumber).
-		Select()
+	query := p.getDb().Model(&pgWorkflows)
+	for key, value := range filters {
+		if value != "" {
+			query = query.Order(key + " " + value)
```

Comment:
Read about sql injection and see how you can prevent it in go lang? Are we susceptible to SQL injection here? Can we validate?

Reply:
you are using filters for ordering? Shouldn't filters be used for filtering and then sortBy query params for ordering? This doesn't feel right.

Reply:
I was thinking about not calling it filters. I will rename it to something else.

As far as SQL injection is concerned, the parser ensures no illegal filters passed